lib/src/file_picker.dart

```diff
@@ -162,6 +162,9 @@ abstract class FilePicker extends PlatformInterface {
   /// stay in front of the Flutter window until it is closed (like a modal
   /// window). This parameter works only on Windows desktop.
   ///
+  /// [mimeType] can be optionally set to provide a mime type for the file.
```

Comment:
This argument somewhat conflicts with `FileType` when the FileType is any or custom.
Can we instead get the mime type from the file bytes or name?

This also only does something for Android in your PR, despite claiming that it works on iOS also.